tests/ryzenai/test_configuration.py

```diff
@@ -0,0 +1,173 @@
+import unittest
+
+from parameterized import parameterized
+
+from optimum.amd.ryzenai import AutoQuantizationConfig, ExtraOptions, QuantizationConfig
+
+
+class TestExtraOptions(unittest.TestCase):
```

Comment:
Are the new quantization parameters tested?

Reply:
In CI?